barretenberg/ts/src/barretenberg_api/index.ts

```diff
@@ -1229,6 +1229,6 @@ export class BarretenbergApiSync {
       outTypes.map(t => t.SIZE_IN_BYTES),
     );
     const out = result.map((r, i) => outTypes[i].fromBuffer(r));
-    return out as any;
+    return out[0];
```

Comment:
maybe I forgot to test this function through wasm? This function isn't tested through CI, so its possible that `out as any` was wrong.